language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config29.json

```diff
@@ -31,7 +31,7 @@
       "insertTextFormat": "Snippet"
     },
     {
-      "label": "version",
+      "label": "'version",
```

Comment:
Seems like the original issue complains about the `insertText`, not the label. And I think the insertText has been fixed by another PR. Which means we don't have to do anything new to fix this.

Reply:
Just saw your comment on the issue regarding https://github.com/ballerina-platform/ballerina-lang/pull/38003. This is fine then.